be/src/vec/json/json_parser.h

```diff
@@ -49,8 +49,9 @@ Field getValueAsField(const Element& element) {
         return Field::create_field<TYPE_BIGINT>(element.getInt64());
     }
     // doris only support signed integers at present
+    // use largeint to store unsigned int64
     if (element.isUInt64()) {
-        return Field::create_field<TYPE_BIGINT>(element.getInt64());
```

Comment:
can we handle TYPE_BOOLEAN in Field so that we can get the accurate boolean type